namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/00-namespace.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: court-probation-e2e
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "dev"
+    cloud-platform.justice.gov.uk/namespace: "court-probation-e2e"
```

Comment:
Is it much easier/less-complex to use the existing court-probation-* namespace? @jhevans

Reply:
It is easier to maintain with the new namespace. But I go with the consensus.

Reply:
From @jhevans